content/css/concepts/colors/terms/rgba/rgba.md

```diff
@@ -51,3 +51,5 @@ h2 {
   color: rgba(0, 0, 0, 25%);
 }
 ```
+
```

Comment:
Add in a sentence here, that connects the above code blocks to the output. You can explain what is happening in the code that results in the output image you added.